ui/css/logviewer.css

```diff
@@ -114,17 +154,6 @@ body {
     border-color: #d58512;
 }
 
-.logviewer-actionbar {
-    min-width: 410px;
-}
-
-.logviewer-actionbtn {
-    display: inline-block;
-    margin: 8px 8px 8px 0px;
-    padding: 6px 8px;
-    border: 1px solid #dddddd;
-}
-
 .logviewer-actionbtn span {
```

Comment:
There's still a few `.logviewer-(thing)` classes in logviewer.css and I plan to convert them all to `lv-(thing)` for brevity/consistency in a subsequent PR.